src/Audit/Display/AbstractAuditLogDisplay.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+/*
+ * This file is part of Packagist.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *     Nils Adermann <naderman@naderman.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Audit\Display;
+
+use App\Audit\AuditRecordType;
+
+abstract readonly class AbstractAuditLogDisplay implements AuditLogDisplayInterface
+{
+    public function __construct(
+        public \DateTimeImmutable $datetime,
+        public ActorDisplay $actor,
+    ) {
+    }
+
+    abstract public function getType(): AuditRecordType;
+
+    public function getDateTime(): \DateTimeImmutable
+    {
+        return $this->datetime;
+    }
+
+    public function getActor(): ActorDisplay
+    {
+        return $this->actor;
+    }
+
+    public function getTypeTranslationKey(): string
```

Comment:
instead of returning a translation key, I suggest returning a `TranslatableInterface` (probably using the `TranslatableMessage` implementation available in core in each case).

The benefits are:
1. it will make it possible to use placeholders if needed
2. when not using dynamic keys (unlike what it does here), keys can be extracted by the tooling (as it is aware that the constructor of TranslatableMessage takes a translation key)